src/site/content/en/metrics/fcp/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: First Contentful Paint (FCP)
+authors:
+  - philipwalton
+date: 2019-11-07
+updated: 2019-11-07
+description: |
+  This post introduces the First Contentful Paint (FCP) metric and explains
+  how to measure it
+tags:
+  - performance
+  - metrics
+---
+
+{% Aside %}
+  First Contentful Paint (FCP) is an important, user-centric metric for
+  measuring [perceived load speed](/user-centric-metrics/#types-of-metrics)
+  because it marks the first point in the page load timeline that the user can
+  see anything on the screen—a fast FCP helps reassure the user that something
+  is [happening](/user-centric-metric/#user-centric-metric-questions).
```

Comment:
404 link, ditto other similar links in other pages (is it because it's staging?)

Reply:
Yep this has been fixed in a subsequent commit.